src/tests/EdFi.DataManagementService.Tests.E2E/Features/Descriptors/ReadDescriptorsValidation.feature

```diff
@@ -103,24 +67,6 @@ Feature: Read a Descriptor
                   []
                   """
```

Comment:
Comment from @stephenfuqua : @msilesgap when we merge this PR, some of the scenarios will move. We will need to update the spreadsheet accordingly.

Why? Suganya and I discussed and agreed that this would simplify our ability to only reset OpenSearch between tests if really needed. Thus we want to keep all of the OpenSearch-related tests together in one place. We thought about using tags, but there is also a setup step that needs to happen ("background"), and we did not want that setup step scattered in many different places.